### DISCUSSION TITLE
Permeability difference between two samples

### DISCUSSION THREAD

**Discussion by Fioti38** [PMEAL/OpenPNM#2480] [Q&A]
Hello ,

I 've done a simulation to compute permeability on two different pore networks extracted from two samples (of different porosities : 3.6% and 19%) . Experimentally, for these 2 samples Im getting a permeability with a difference of 2 order of magnitude.

In my simulation in openPNM,the difference is only  4x between both permeabilities, which isn't representative regarding their porosities.
 I've done a comparison between geometrical parameters from both samples and found out that this 4x factor appears at the mean diagonal of A matrix (of hydraulic conductivities). I don't know what to investigate further to understand how to get closer to my experimental results and take into account in my simulations that these two samples are so different in porosities ?
Below are images from my comparison :
![image](https://user-images.githubusercontent.com/104492138/173101280-e9790c72-f69a-46a0-bada-431be4ce0c85.png)

Merci beaucoup

**Comment by Fioti38** [comment]
Hello , anyone has  hint on this theoretical problematic ? Thank you all

**Reply by jgostick** [reply]
Sorry for not replying...I'm just swamped right now.

**Reply by Fioti38** [reply]
Yes of course, I understand, thank you professor
